lib/grpc/server/supervisor.ex

```diff
@@ -48,7 +48,8 @@ defmodule GRPC.Server.Supervisor do
     check_deps_version()
 
     children =
-      if Application.get_env(:grpc, :start_server, false) do
+      if (Keyword.has_key?(opts, :start_server) and opts[:start_server]) or
+           Application.get_env(:grpc, :start_server) do
```

Comment:
What is plan to remove global configuration?

Reply:
This change removes the need to deprecate the global config :)

If the key is present in the options, it will have precedence (regardless of the value)

Reply:
Got it. 🙇